### PR TITLE
added instructions for creating static build

### PR DESCRIPTION
**Pull request by AronBuzogany**
Fixes PR #61

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,26 @@ npm start
 
 Which will start the web application
 
+## Static build
+
+If you want a static build of the application you can execute
```
Comment (pheyvaer) [inline README.md:23]: "can" is filler word here. Remove it.

File: README.md [hunk 2]
```diff
@@ -18,6 +18,26 @@ npm start
...
+
+```bash
+npm install -g serve
+serve -s build
+```
+
+Which will either host the website on port 3000 or pick a random port if 3000 is already in use.
+If you want to specify a port you can use the `-l` flag proceeded by the port number of your choice. 
+
+
```
Comment (pheyvaer) [inline README.md:40]: Remove this because this has nothing to do with how to build the app.

**Review by pheyvaer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AronBuzogany) [reply README.md]: Removed it in [commit 402db13](https://github.com/SolidLabResearch/generic-data-viewer/pull/62/commits/402db139419551cb472940c3ea22d759cad0df79)

File: README.md (hunk 2 above)
Comment (AronBuzogany) [reply README.md]: Removed it in [commit 402db13](https://github.com/SolidLabResearch/generic-data-viewer/pull/62/commits/402db139419551cb472940c3ea22d759cad0df79)

**Review by pheyvaer — APPROVED** [review]

**Pull request merged**